crates/holochain/src/core/state/cascade/network_tests.rs

```diff
@@ -446,6 +446,7 @@ async fn get_links_from_another_agent() {
         target: target_entry_hash.clone(),
         timestamp: links.get(0).unwrap().timestamp.clone(),
         tag: link_tag.clone(),
+        create_link_hash: link_add_hash.clone(),
```

Comment:
What if, by convention, we simply defined the "hash of a Link" as the hash of the CreateLink element that created it? Then we can just call this "hash".

```suggestion
        hash: link_add_hash.clone(),
```

Reply:
I'm not super keen on this because the hash really points to the header. It might be confuse people into thinking it points to the link which is actually a derived value